src/test/lib/render_test.ts

```diff
@@ -1057,7 +1057,7 @@ suite('render()', () => {
       assert.equal(stripExpressionMarkers(container.innerHTML), '<div></div>');
 
       // Wait for mutation callback to be called
-      await new Promise(setTimeout);
+      await new Promise((resolve) => setTimeout(resolve));
```

Comment:
Promise became stricter or something, so now the arguments are strongly typed. because setTimeout doesn't take a `reject`, it failed before